docs/source/release/v6.6.0/direct_geometry.rst

```diff
@@ -10,31 +10,38 @@ General
-- The 'Update Estimate' button has been removed in ALFView. An estimate value for the peak centre is automatically re-calculated whenever the tube selection changes.
-- An option to export the right hand side plot to a workspace has been added (top right).
-- An option to open the right hand side plot in a new window to allow more plotting customizations has been added (top right).
-- An option to reset the extracted data in the right hand side plot has been added (top right).
-- The average two theta value is now displayed in ALFView when a tube is selected (bottom right).
-- The number of selected tubes is now displayed in ALFView when the tube selection is changed (bottom right).
-- The FunctionBrowser widget has been replaced with a box to specify the peak centre of a Flat Background + Gaussian in ALFView.
-- The 'Fit' button will now trigger the calculation of the Rotation angle, if the Fit is successful.
-- Updated the documentation for the ALFView interface.
-- :ref:`LagrangeILLReduction <algm-LagrangeILLReduction>` now allows the user to select normalisation approach, loads time and temperature metadata from the ASCII files, and properly handles interpolation and energy range of the water correction depending on the user's choice of `UseIncidentEnergy` property value.
+- The ability to load an optional vanadium run into :ref:`ALFView <ALFView-ref>` for normalisation is now possible. This vanadium run is saved between sessions.
+- The tools in the :ref:`ALFView <ALFView-ref>` Pick tab which are not useful for the main workflow have been hidden.
+- The ``Select whole tube`` tool on the :ref:`ALFView <ALFView-ref>` Pick tab can now be used to select individual tubes.
+- The ``Draw a rectangle`` tool on the :ref:`ALFView <ALFView-ref>` Pick tab can now be used to select one or more neighbouring tubes.
+- The ``Edit a shape`` tool on the :ref:`ALFView <ALFView-ref>` Pick tab can now be used to select, move, resize and delete highlighted tubes.
+- The ``Update Estimate`` button has been removed in :ref:`ALFView <ALFView-ref>`. An estimate value for the peak centre is automatically re-calculated whenever the tube selection changes.
+- An option to export the right hand side plot to a workspace has been added to :ref:`ALFView <ALFView-ref>`.
+- An option to open the right hand side plot in a new window to allow more plotting customizations has been added to :ref:`ALFView <ALFView-ref>`.
+- An option to reset the extracted data in the right hand side plot has been added to :ref:`ALFView <ALFView-ref>`.
+- The average two theta value is now displayed in :ref:`ALFView <ALFView-ref>` when a tube is selected.
+- The number of selected tubes is now displayed in :ref:`ALFView <ALFView-ref>` when the tube selection is changed.
+- The FunctionBrowser widget has been replaced with a box to specify the peak centre of a Flat Background + Gaussian in :ref:`ALFView <ALFView-ref>`.
```

Comment:
Is this not a duplicate of the first bullet?

Reply:
Yes Good catch